delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -244,4 +244,11 @@ def get_all_pools(self):
 
     def clear_alert(self, sequence_number):
         url = "/alarm/currentalarm?sequence=%s" % sequence_number
-        return self.call(url, method="DELETE", log_filter_flag=True)
+
+        # Result always contains error code and description
+        result = self.call(url, method="DELETE", log_filter_flag=True)
+        if result['error']['code']:
+            msg = 'Clear alert failed with reason: %s.' \
+                  % result['error']['description']
+            raise exception.InvalidResults(msg)
```

Comment:
Got your point, as the error code differ from different backend, went for description. Anyway, we are doing some update about backend exception handling, I think this can be chaged to follow that and improve

Reply:
Ok. You can simply propagate the error code.
Lets plan it. Open an issue